storage/series/series.go

```diff
@@ -180,8 +185,18 @@ func (s *dbSeries) updateBlocksWithLock() updateBlocksResult {
 			case CacheRecentlyRead:
 				sinceLastRead := now.Sub(currBlock.LastReadTime())
 				shouldUnwire = sinceLastRead >= wiredTimeout
+			case CacheLRU:
+				// The tick is responsible for managing the lifecycle of blocks that were not
+				// read from disk (not retrieved), and the WiredList will manage those that were
+				// retrieved from disk.
+				if !currBlock.WasRetrievedFromDisk() {
+					shouldUnwire = true
+				}
+			default:
+				panic(fmt.Sprintf("Unhandled cache policy in series tick: %s", cachePolicy))
```

Comment:
nit: Use lower case first letter for panic to match logs, etc.

Reply:
replaced with a call to log.fatal